metrics/metrics.go

```diff
@@ -65,6 +65,18 @@ var (
 		Help: "Duration between time round received and time round expected.",
 	})
 
+	// ClientHTTPHeartbeatLatency measures the latency of HTTP hearbeat andomness requests.
+	ClientHTTPHeartbeatLatency = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
why does this need to be separated out from the existing latency tracked at the http level? I think this will be the same as that metric

Reply:
From what I can tell, they measure different things: HTTPLatency seems to measure latency of the HTTP API call (not 100% sure). ClientHTTPHeartbeatLatency measures latency of the randomness.

ClientHTTPHeartbeatLatency was requested by clients (filecoin) so they can know when the specific HTTP CDNs that they use are healthy. So, I realize, each measurement has to also be labelled with the HTTP address that it is hitting.